.prettierrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  trailingComma: "es5",
```

Comment:
What does es5 signify here?
When all will it add tralingComma

Reply:
please read this section for reference on `trailingComma: "es5",` https://prettier.io/docs/en/options.html#trailing-commas